pages/index.vue

```diff
@@ -1,17 +1,42 @@
 <template>
 
-  	<b-container>
+	<b-container>
 
 		<h1>Origami Annotation Tool</h1>
+
+		<!-- Debug component - shows file contents -->
+		<b-row>
+			<h2>TSV File</h2>
+			<textarea rows="5" cols="200" v-model="stringifiedTsvFile"></textarea>
+		</b-row>		
+
+		<!-- Selects participant.tsv file -->
+		<file-selector 
+			content-type="text/tab-separated-values"
+			v-on:file-selected="saveTsvFileData($event)">
+		</file-selector>
+
+		<!-- Debug component - shows file contents -->
 		<b-row>
-		    <textarea :rows="fileTextLines" cols="300">
-				{{ this.fileText.join("\n") }}
-			</textarea>
+			<h2>JSON File</h2>
```

Comment:
Let's call this `Data dictionary` for now. Later we'll add some explanations.

Reply:
This change will be added in the a forthcoming PR.